packages/data-point/lib/entity-types/validate-modifiers/validate-modifiers.js

```diff
@@ -0,0 +1,38 @@
+const Util = require('util')
+const _ = require('lodash')
+
+function validateProperties (id, spec, validKeys) {
+  const differentKeys = _.difference(Object.keys(spec), validKeys)
+  if (differentKeys.length > 0) {
+    throw new Error(
+      Util.format(
+        'Entity "%s" did not recognize the following properties: %s, valid properties are: %s.\nPlease review your entity and make any corrections so it can be parsed:\n\'%s\': %s',
```

Comment:
can you add a new line after `Entity "%s" did not recognize the following properties: %s`? so there is a line just for listing the valid properties